### PR TITLE
fix(vue): handle vmodel directive

### PR DESCRIPTION
**Pull request by Shyrro**
Signed-off-by: Shyrro <zsahmane@gmail.com>

Add support for `v-model` directive

### CODE REVIEW

**Comment by TimKolberger** [comment]
Thank you for working on this @Shyrro ✨ 
PR looks good to me, and I am down to merge this when the typecheck is successful in the CI. Could you take a look?

File: packages/vue/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@testing-library/vue": "6.6.1",
     "@types/jsdom": "20.0.1",
     "@types/testing-library__jest-dom": "5.14.5",
+    "@vitejs/plugin-vue": "^4.0.0",
```
Comment (TimKolberger) [inline packages/vue/package.json:38]: **suggestion**: Let's pin the devDeps to ensure reproducible builds. We use renovate to schedule those updated automatically.
Kindly update the lock file too.

```suggestion
    "@vitejs/plugin-vue": "4.0.0",
```

File: packages/vue/test/sfc.test.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { describe, expect, it } from 'vitest'
+import { screen, render, fireEvent } from '@testing-library/vue'
+import PolyTest from './poly-test.vue'
+
+describe('SFC related features', () => {
```
Comment (TimKolberger) [inline packages/vue/test/sfc.test.ts:5]: **praise (non-blocking)**: love the test suite!

**Review by TimKolberger — COMMENTED** [review]
Awesome work @Shyrro Thanks for working on that!

To merge and release this we need a changeset file in the PR branch.
Kindly run 'pnpm changeset' in the repo root, choose vue with a minor bump and describe the changes as you would expect them in a changelog.  

You can read more about the process here: https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

Or reach out to me, and we can do it together.

**Comment by Shyrro** [comment]
> Awesome work @Shyrro Thanks for working on that!
> 
> To merge and release this we need a changeset file in the PR branch. Kindly run 'pnpm changeset' in the repo root, choose vue with a minor bump and describe the changes as you would expect them in a changelog.
> 
> You can read more about the process here: https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md
> 
> Or reach out to me, and we can do it together.

Thanks for your review @TimKolberger ! Will try to update to changelog today !

**Comment by Shyrro** [comment]
Hi @TimKolberger ! 

I just updated the changelog + updated the lock file, can you please check if everything is okay ?

Thanks !

**Pull request merged**